ts/components/dialog/debug/ReleaseChannel.tsx

```diff
@@ -0,0 +1,95 @@
+import { capitalize } from 'lodash';
+import { useDispatch } from 'react-redux';
+import useUpdate from 'react-use/lib/useUpdate';
+import type { Dispatch } from 'redux';
+import { localize } from '../../../localization/localeTools';
+import { updateConfirmModal } from '../../../state/ducks/modalDialog';
+import { Flex } from '../../basic/Flex';
+import { SessionButtonColor } from '../../basic/SessionButton';
+import { SessionRadioGroup } from '../../basic/SessionRadioGroup';
+import { HintText } from '../../basic/Text';
+import { ALPHA_CHANNEL, LATEST_CHANNEL, type ReleaseChannels } from '../../../updater/types';
+import { Storage } from '../../../util/storage';
+
+const changeReleaseChannel = (
+  channel: ReleaseChannels,
+  dispatch: Dispatch,
+  forceUpdate: () => void
+) => {
+  window.log.debug(
+    `[debugMenu] Setting release channel to ${channel}. It was ${Storage.get('releaseChannel') || 'not set'}`
+  );
+  dispatch(
+    updateConfirmModal({
+      title: localize('warning').toString(),
```

Comment:
seeing what this does, maybe this should be a hook that returns a function (just so it would have access to dispatch, and no need to forward it.
It's just that it looks like a hook, but has no use prefix.

Reply:
Fixed